service/controller/key/key.go

```diff
@@ -111,7 +111,7 @@ const (
 )
 
 const (
-	KubernetesNetworkSetupDocker = "0.1.0"
+	KubernetesNetworkSetupDocker = "0.2.0"
```

Comment:
I am fine with this - @tfussell do you want to include this in `12.0.0` or would you rather get it in `12.0.1`?

Reply:
I reckon it can go in 12.0.0 since I'm going to try to fix the flannel bridge routing issue before releasing. I don't expect this to change anything for KVM.

Reply:
Yeah this is mostly for AWS, and this change is just aligning version

But I tested cluster creation and it was fine.